pycbc/events/stat.py

```diff
@@ -1561,6 +1563,12 @@ def rank_stat_coinc(self, s, slide, step, to_shift,
 
         # First get signal PDF logr_s
         stat = {ifo: st for ifo, st in s}
+
+        try:
+            self.curr_mchirp = kwargs['mchirp']
+        except KeyError:
+            pass
```

Comment:
The setting of `self.curr_mchirp` is needed for the stat to run in Live but this code is also called by `pycbc_coinc_findtrigs` where the object being called doesn't have the keyword argument 'mchirp'.

Therefore I've used a try except with a simple `pass` because the error will be expected.

Reply:
Ok, see my other suggestion on this though - a simple `if 'mchirp' in kwargs` is simpler (and might even be faster) than raising and catching an exception.